job-search-react/src/components/JobApp.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import Typed from 'react-typed'
+
+export default function JobApp() {
+  return (
+<div>
+    <div class="container">
+        <h1 class="main-header" id="main-header">JOB SEARCH</h1>
+        <div class="typed-container">
+            <div id="typed-strings"><p class="lead-in">#VetsWhoCode <span id="typed" class="typed"> <Typed strings={['provides job placement assistance.','makes a difference in the lives of veterans.']}typeSpeed={50} backSpeed={50} loop /></span>
```

Comment:
This was really hard to find `Typed`. Could you please use Prettier to format this file for more easy searching.

Reply:
Prettiered the hell out of it!